profiles/product_with_manufacturer.yaml

```diff
@@ -15,24 +15,32 @@ sort:
 mappings:
   - file_column: "id"
     entity_path: "id"
+    field_type: "uuid"
```

Comment:
I'm actually not sure if we should add these here in the profiles or the validation (for the values in the csv) should just be driven by the types in the schema. WDYT? But added it here for now nonetheless just for some practice.

Reply:
Ah that was also my question what's the point of it. So you want to use this information for potentially validating values?

In that case it might be better to gather these types programmatically from our entity schema and preparing these for easy lookup during the data transformations. So we might actually want to have mappings defined in the schema / profile files and mappings we use at runtime with additional information like the `field_type` 🤔 .

But I'm still not sure if we need to do all the type validation work ourselves or just rely on the Shopware API and response for that to not bloat this codebase too much 🤷‍♂️

Reply:
Not needed indeed. We can rely only on the types from the schema. This was more a remnant of my previous solution where this type has driven which serializer/deserializer to use. 

But I think validation should be done also on our part, not just left to the API. Because we will need some deserialization and type casting before sending the data through to the API, which can lead to some unwanted values being sent, for example casting a non-numeric string to int can result in 0 where we actually don't want to send 0 as value. We had this type of ticket also for the current import/export. Or do you have a better idea to handle this?